backend/sozisel/lib/sozisel_web/schema/resolvers/participant_resolvers.ex

```diff
@@ -186,4 +191,88 @@ defmodule SoziselWeb.Schema.Resolvers.ParticipantResolvers do
 
     verify_launched_event(ctx, launched_event_id, on_verify)
   end
+
+  def submit_whiteboard_result(
+        _parent,
+        %{
+          input: %{
```

Comment:
do this inside function body so the function line is not that long

Reply:
done